### DISCUSSION TITLE
1.10.1

### DISCUSSION THREAD

**Discussion by jwallet** [jwallet/spy-spotify#381] [Release]
- Fixed Spotify API Access, was sending too many requests when using the api for playback status #370 #378  #380 

[![GitHub Releases](https://img.shields.io/github/downloads/jwallet/spy-spotify/1.10.1/total)](https://github.com/jwallet/spy-spotify/releases/download/1.10/Spytify-v.1.10.zip) [![Donate](https://img.shields.io/badge/support-donate-ff69b4)](https://jwallet.github.io/spy-spotify/donate.html) [![FAQ](https://img.shields.io/badge/help-faq-21DA65)](https://jwallet.github.io/spy-spotify/faq.html) [![Sub Reddit](https://img.shields.io/reddit/subreddit-subscribers/spytify.svg?label=r%2Fspytify)](https://www.reddit.com/r/spytify)

<hr /><em>This discussion was created from the release <a href='https://github.com/jwallet/spy-spotify/releases/tag/1.10.1'>1.10.1</a>.</em>

**Comment by thewhitegrizzli** [comment]
@jwallet if you are looking for help I'd be happy to jump in and support the project.

**Comment by HomeyDawg** [comment]
@jwallet,   Great work with this project.  Software works really well, simple to use, great results.   
Tried to give you a donation, but clicking on the Paypal link returns:
We can’t find this profile
Make sure the link’s right and profile hasn’t been turned off.
-- Is this intentional on your part?  TYVM.

**Reply by jwallet** [reply]
Yes, Paypal was a pain for donation. They locked my account because I was receiving donations and my business profile was incomplete. Since I don't want to provide more info about me, I only use Stripe services through Donorbox now.

**Comment by funkydiver** [comment]
Not sure if this is the same for everyone... however, when trying to change folder (and I've updated the source files) I constantly get the following... it'll only choose desktop and won't let me browse folders. 

![image](https://user-images.githubusercontent.com/63501641/198723252-e0b78768-5934-4757-a282-029281264fb5.png)
